src/mutmap/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+find_package(Rscript)
+
+set(MUTMAP_DIR ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBEXECDIR}/mutmap)
+
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/mutmap.sh.in"
+  "${CMAKE_CURRENT_BINARY_DIR}/mutmap.sh" @ONLY)
+
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/dist/build.R"
+  "${CMAKE_CURRENT_BINARY_DIR}/build.R" @ONLY)
+
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/dist/bundle.js"
+  "${CMAKE_CURRENT_BINARY_DIR}/bundle.js" @ONLY)
+
```

Comment:
The biggest reason currently is that I'm using browserify, which handles bundling all the different javascript source files together. But it also allows you to use the `require `syntax, which is very nice for using js in a modular way. Without it we would need to do all of the bundling ourselves. I can give it a shot and see how ugly it turns out, but I don't think it's worth it. Once you have node, there are tons of useful tools at your disposal.  It's just how things are done in js land. For example, I'd like to convert all my OOP code to typescript when I have a couple extra hours, to make it more maintainable in the future. Using typescript requires node.